### PR TITLE
Document adding a route on non-Weave hosts

### PR DESCRIPTION
**Pull request by bboreham**
ping @abuehrle for doc review

### CODE REVIEW

File: site/using-weave/service-management.md [hunk 1]
```diff
@@ -38,6 +39,17 @@ With the above in place, you are ready to connect to the 'nc' service from anywh
 
 Similar NAT rules to the above can used to expose services not just to the outside world but also other, internal, networks.
 
+###<a name="export-route"></a> Exporting Services to the wider network
+
+If you want hosts that are not running Weave Net to access services
+exposed as above, on each of those machines:
```
Comment (rade) [inline site/using-weave/service-management.md]: No, that's not what this does. The "above" section is making *individual* services accessible to the host network *without altering the network configuration* - individual services end up with an IP/port on the host network.

By contrast, in this section we show how to make *the entire weave network* (or subnets thereof) accessible. Alas it *does* require networking changes (namely the addition of a route).

NB: a prerequisite here is that the subnet in use by weave is 'free' not just on the weave peers (as normal) but also all the hosts that require access to the weave network.

Also, position wise I reckon it's confusing for this to sit between the exporting/importing sections. I'd stick it at the very end.

Comment (abuehrle) [inline site/using-weave/service-management.md]: Should also use title case for the heading, Exporting Services to the Wider Network  -- rule is to capitalize words with four or more letters.

File: site/using-weave/service-management.md [hunk 2]
```diff
@@ -38,6 +39,17 @@ With the above in place, you are ready to connect to the 'nc' service from anywh
...
+
+    ip route add <network-cidr> via <exposing-host>
+
+Where `<network-cidr>` gives the IP address range used by weave,
+e.g. `10.2.0.0/16` or `10.32.0.0/12` and `<exposing-host>` is the
+address of the machine that has run `weave expose`, e.g. `host2` in
+the example above.
```
Comment (abuehrle) [inline site/using-weave/service-management.md]: Where, 

* `<network-cidr> is the IP address range that is used by Weave, for example, 10.2.0.0/16` or `10.32.0.0/12` and, 
* `<exposing-host>` is the address of the machine on which `weave expose` was run. For example, `host2` in the example above <-- not sure which example this is referring to?

**Comment by rade** [comment]
Q4 in https://github.com/weaveworks/weave/blob/master/site/faq.md needs updating.

File: site/using-weave/service-management.md (hunk 1 above)
Comment (bboreham) [inline site/using-weave/service-management.md]: @rade the previous section does make the entire weave network accessible, from the local host.  But I get your point: my addition is not about "services exposed as above".

**Comment by bboreham** [comment]
Updated; PTAL

File: site/using-weave/service-management.md [hunk 3]
```diff
@@ -10,6 +10,7 @@ This section contains the following topics:
  * [Binding Services](#binding)
  * [Routing Services](#routing)
  * [Dynamically Changing Service Locations](#change-location)
+ * [Exposing the Weave Network More Widely](#export-route)
```
Comment (rade) [inline site/using-weave/service-management.md]: Looking at the current doc structure, I don't think this is the right place for this. It is not about services and instead is more like the "Integrating with the Host Network" section which lives at the same level as this entire doc. So I reckon it should either go into that (and that section should be renamed) or be a separate section at the same level.

@abuehrle what do you think?

File: site/using-weave/service-management.md [hunk 4]
```diff
@@ -106,6 +107,25 @@ Furthermore, as explained in Binding Services, service locations can be dynamica
 For example, you can move the netcat service to `$HOST4:2211`  and it will retain its 10.2.1.3:3322 endpoint on the Weave network.
 
 
+###<a name="export-route"></a>Exposing the Weave Network More Widely
+
+You can use Linux routing to give access to the Weave network from
```
Comment (rade) [inline site/using-weave/service-management.md]: I think it's "Weave Network" @abuehrle?

File: site/using-weave/service-management.md [hunk 5]
```diff
@@ -106,6 +107,25 @@ Furthermore, as explained in Binding Services, service locations can be dynamica
...
+hosts that are not running Weave. First, ensure that the [IP
+subnet](site/how-it-works/ip-addresses.md) used by Weave does not
+clash with anything on those other hosts, then on one Weave host run:
```
Comment (rade) [inline site/using-weave/service-management.md]: I believe we always say "Weave Net", not just "Weave". @abuehrle?

File: site/using-weave/service-management.md [hunk 6]
```diff
@@ -106,6 +107,25 @@ Furthermore, as explained in Binding Services, service locations can be dynamica
...
+
+    weave expose
+
+Then on each non-Weave machine that you want to access the network,
+run:
+
+    ip route add <network-cidr> via <exposing-host>
+
+Where `<network-cidr>` gives the IP address range used by weave,
```
Comment (rade) [inline site/using-weave/service-management.md]: I would make "IP address range used by Weave Net" a hyperlink to (the .md of) https://www.weave.works/documentation/net-1-5-0-ipam/#range

Comment (bboreham) [inline site/using-weave/service-management.md]: On further thought, this doesn't have to map to the IPAM range, just to some subnet in use on Weave Net that you want to route.

**Comment by bboreham** [comment]
I have moved the whole section to `ipam.md` as suggested, fixed the "Weave Net" usage, squashed and rebased.

File: site/using-weave/host-network-integration.md [hunk 7]
```diff
@@ -38,6 +38,20 @@ Exposed addresses can also be added to weaveDNS by supplying fully qualified dom
     host2$ weave expose -h exposed.weave.local
     10.2.1.132
 
+###<a name="routing"></a>Routing from another host
+
```
Comment (abuehrle) [inline site/using-weave/host-network-integration.md]: Title case for the title.  This means only using capital letters for the principal words. Articles, conjunctions, and prepositions do not get capital letters unless they start the title.

File: site/using-weave/host-network-integration.md [hunk 8]
```diff
@@ -38,6 +38,20 @@ Exposed addresses can also be added to weaveDNS by supplying fully qualified dom
...
+Having run `weave expose` as above, you can use Linux routing to give
+access to the Weave Network from hosts that are not running Weave Net:
+
```
Comment (abuehrle) [inline site/using-weave/host-network-integration.md]: After running `weave expose`, you can use Linux routing to provide access to the Weave network from hosts that are not running Weave Net:

File: site/using-weave/host-network-integration.md [hunk 9]
```diff
@@ -38,6 +38,20 @@ Exposed addresses can also be added to weaveDNS by supplying fully qualified dom
...
+    ip route add <network-cidr> via <exposing-host>
+
+Where `<network-cidr>` is an IP address range in use on Weave Net,
+e.g. `10.2.0.0/16` or `10.32.0.0/12` and `<exposing-host>` is the
+address of the machine where you ran `weave expose`.
+
```
Comment (abuehrle) [inline site/using-weave/host-network-integration.md]: Where,
 * `<network-cidr>` is an IP address range in use on Weave Net,
for example,  `10.2.0.0/16` or `10.32.0.0/12` and, 
 * `<exposing-host>` is the address of the machine on which you ran `weave expose`.

**Comment by bboreham** [comment]
Updated per comments.

File: site/using-weave/host-network-integration.md [hunk 10]
```diff
@@ -38,6 +38,21 @@ Exposed addresses can also be added to weaveDNS by supplying fully qualified dom
...
+    ip route add <network-cidr> via <exposing-host>
+
+Where,
+ * `<network-cidr>` is an IP address range in use on Weave Net,
+for example,  `10.2.0.0/16` or `10.32.0.0/12` and,
+ * `<exposing-host>` is the address of the machine on which you ran `weave expose`.
+
+>**Note:** You must ensure that the [IP subnet](site/how-it-works/ip-addresses.md)
+used by Weave Net does not clash with anything on those other hosts.
```
Comment (rade) [inline site/using-weave/host-network-integration.md]: The obvious question a user will have at that point is... "How?". I suggest we turn some of the above test into a hyperlink to ipam.md#range.

**Comment by bboreham** [comment]
Updated the FAQ and "subnet used by Weave Net", and squashed.

**Pull request merged**